src/main/java/Parser.java

```diff
@@ -0,0 +1,102 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Parser {
+    private static final String HORIZONTAL_LINE = "  ____________________________________________________________";
+
+    public static void handleCommand(String command, ArrayList<Task> storedTasks) throws DukeException {
+        if (command.equals("bye")) {
+            System.out.println(HORIZONTAL_LINE + "\n  Bye. Hope to see you again soon!\n" + HORIZONTAL_LINE);
+        } else if (command.equals("list")) {
+            System.out.println(HORIZONTAL_LINE);
+            for (int i = 0; i < storedTasks.size(); i++) {
+                System.out.println("  " + String.valueOf(i + 1) + ". " + storedTasks.get(i));
```

Comment:
Should this be extracted out to the Ui class? I noticed this in the other lines too.